spec/models/trip_spec.rb

```diff
@@ -1,5 +1,88 @@
 require 'rails_helper'
 
 RSpec.describe Trip, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+  context '#trips' do
+    it 'has a name' do
+      trip = create_trip
+
+      expect(trip.name).to eq("Machu Picchu")
+    end
+
+    it 'has an trip cost' do
+      trip = create_trip
+      expect(trip.trip_cost).to eq(5000.00)
+    end
+
+    it 'has an total cost' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.